components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.common/src/main/java/org/wso2/carbon/identity/api/server/idp/common/Constants.java

```diff
@@ -116,6 +116,16 @@
         ERROR_CODE_OUTBOUND_PROVISIONING_CONFIG_NOT_FOUND("60029", "Unable to update Outbound " +
                 "Provisioning Connector", "Outbound Provisioning Connector properties have not specified " +
                 "for connector : %s"),
+        ERROR_CODE_INVALID_USER_CLAIM_URI("60030", "Invalid user ID claim URI",
+                "User ID claim URI: %s does not match with the claim mappings"),
+        ERROR_CODE_INVALID_ROLE_CLAIM_URI("60031", "Invalid role claim URI",
+                "Role claim URI: %s does not match with the claim mappings"),
+        ERROR_CODE_NOT_EXISTING_CLAIM_URI("IDP-60032", "Invalid claim URI",
+                "One or more local claim URIs does not exist"),
```

Comment:
grammer :  URIs does not exist

Reply:
Fixed in f44b32e0efda15349b0bbf3d014763ee0b8297ad